pkg/trace/api/otlp.go

```diff
@@ -473,6 +473,25 @@ func setMetricOTLP(s *pb.Span, k string, v float64) {
 	}
 }
 
+// Default keys used to find a value for _dd.peer.service.source, applied in the order defined below.
```

Comment:
In Go, documentation starts with the identifier name, so:
```
// peerServiceDefaults specifies ...
```

Reply:
Updated